src/global.scss

```diff
@@ -46,7 +46,7 @@ a:hover {
 .section {
   background-color: $section-bg;
   box-sizing: border-box;
-  min-height: calc(100vh - #{$navbar-height});
+  min-height: calc(100vh - #{$navbar-height} + 1px);
```

Comment:
why +1?

Reply:
ignore this, it was cause the border for the next section was showing up when the previous section filled the screen, but I think a better fix is switching from `border` to `outline`